src/site/content/en/blog/window-controls-overlay/index.md

```diff
@@ -317,6 +317,25 @@ for example, `"standalone"`.
 
 {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/1VonSc0jOiOureeSkqtf.png", alt="PWA running in standalone mode with the window controls overlay displayed in the body.", width="800", height="428" %}
 
+## UI considerations
+
+While it may be tempting, it is not recommended to try to create classic dropdown
+menu bars in the Window Controls Overlay area. Doing so would violate the
+[design guidelines on macOS](https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-bar-menus/),
+a platform on which users expect menu bars (that typically include both system-provided menus and
```

Comment:
```suggestion
a platform on which users expect menu bars (that typically includes both system-provided menus and
```

Reply:
'Includes' seems to refer to 'platform'.

Reply:
In this case it refers to menu bars.